src/towncrier/test/test_build.py

```diff
@@ -1398,6 +1398,58 @@ def test_default_start_string_markdown(self, runner):
 
         self.assertEqual(expected_output, output)
 
+    @with_project(
+        config="""
+        [tool.towncrier]
+        name = "🍫 FooBar"
+
+          [[tool.towncrier.type]]
+          directory = "bugfix"
+          name = "🐛 Bugfixes"
+          showcontent = true
+        """
+    )
+    def test_underline_size_unicode(self, runner):
+        """
+        TODO: Add better description
```

Comment:
Maybe something like this.
```suggestion
        The news file can be generaged for project names,
        and categories that contains emoji in their names.
        This is a test for generating the RST section
        underline to meet the docutils requirements. 
```

Reply:
Whoops, totally forgot to come back around for that. Thoughts on my mypy comment above?